package.json

```diff
@@ -96,12 +97,14 @@
     "uuid": "^3.1.0"
   },
   "resolutions": {
-    "babel-core": "^7.0.0-bridge.0"
+    "babel-core": "^7.0.0-bridge.0",
+    "@types/graphql": "^0.13.1"
```

Comment:
For some weird reason a few of our deps pull in this types package as a runtime dependency, which leads to there being multiple versions… It’s really a bug upstream, but whatevs, can’t fight every battle 🤷‍♂️